examples/todos.roc

```diff
@@ -143,32 +149,14 @@ isSqliteInstalled =
         Ok {} -> Task.ok {}
         Err _ -> Task.err Sqlite3NotInstalled
 
-logRequest : Request -> Task {} *
+logRequest : Request -> Task {} [StdoutErr Str]
 logRequest = \req ->
     datetime = Utc.now! |> Utc.toIso8601Str
 
-    Stdout.line! "$(datetime) $(Http.methodToStr req.method) $(req.url)"
+    Stdout.line "$(datetime) $(Http.methodToStr req.method) $(req.url)"
+    |> Task.mapErr \err -> StdoutErr (Inspect.toStr err)
```

Comment:
Maybe we should keep this as a `StdoutErr Stdout.Err` instead, to be compatible with other `Stdout.Err` handling code.

Reply:
It's an example, nbd